msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -925,14 +927,15 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
 
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
-|`contexts`| Array | 4 | | Specifies the Microsoft 365 application window in which the ribbon customization is available to the user. Each item in the array is a member of a string array. <br>Supported values: `mailRead`, `mailCompose`, `meetingDetailsOrganizer`, `meetingDetailsAttendee`|
+|`contexts`| Array | 6 | | Specifies the Microsoft 365 application window in which the ribbon customization is available to the user. Each item in the array is a member of a string array. <br>Supported values: `mailRead`, `mailCompose`, `meetingDetailsOrganizer`, `meetingDetailsAttendee`, `onlineMeetingDetailsOrganizer`, `logEventMeetingDetailsAttendee`, `default`|
```

Comment:
@Rick-Kirkham, For the `contexts` property, the maxItems in the manifest schema is 7. Request you to please confirm and update.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/a2675fc2-0b02-49b7-b80a-55492bdc31ab)